test/kubectx.bats

```diff
@@ -239,3 +239,26 @@ load common
   [ "$status" -eq 0 ]
   [[ "$output" = "user2@cluster1" ]]
 }
+
+@test "no selected context" {
+  use_config config1
+
+  run ${COMMAND} -c
+  echo "$output"
+  [ "$status" -eq 1 ]
+  [[ "$output" = "error: current-context is not set" ]]
```

Comment:
this message is coming from `kubectl`. we shouldn't test it as we have no control over it.

maybe let's just do this:

`[ "$status" -ne 0 ]` and test that there must be a  failure.

tbh you can just omit this test in this patch. it's not about `-u`.